store/src/main/java/com/book/store/dto/BookDto.java

```diff
@@ -1,15 +1,37 @@
 package com.book.store.dto;
 
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
+
+import jakarta.validation.constraints.Size;
 import lombok.Data;
 
 @Data
 public class BookDto {
     private Long id;
+
+    @NotBlank(message = "Title is required")
```

Comment:
Have you used BookDto validations? I didnt see

Code that are not in use should be removed